crates/picky-krb/src/data_types.rs

```diff
@@ -385,5 +614,257 @@ mod tests {
     }
 
     #[test]
-    fn test_authorization_data() {}
+    fn test_authenticator() {
```

Comment:
General comment: avoid prefixing test functions with `test_`
```suggestion
    fn authenticator() {
```
It’s already in a `tests` module + it only appears when issuing a `cargo test` command + there is already a `#[test]` annotation.

I suggest changing that for all the tests.

Reply:
Done